splice_machine/src/test/java/com/splicemachine/derby/impl/sql/catalog/ViewsInSysIbmIT.java

```diff
@@ -0,0 +1,224 @@
+
+        conn.commit();
+    }
+
+    public static void cleanoutDirectory() {
+        try {
+            File file = new File(getExternalResourceDirectory());
+            if (file.exists())
+                FileUtils.deleteDirectory(new File(getExternalResourceDirectory()));
+            file.mkdir();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @BeforeClass
+    public static void createDataSet() throws Exception {
+        cleanoutDirectory();
+        createData(spliceClassWatcher.getOrCreateConnection());
+    }
+
+    @Test
+    public void testSysColumns() throws Exception {
+        String sqlText = format("select name, tbname, tbcreator, '-'||coltype||'-', nulls, codepage, length, scale, colno, typename, longlength, keyseq from sysibm.syscolumns where tbname='T5' and TBCREATOR='%s'", CLASS_NAME);
+        String expected = "NAME |TBNAME |   TBCREATOR    |     4     | NULLS |CODEPAGE |LENGTH | SCALE | COLNO |TYPENAME  |LONGLENGTH |KEYSEQ |\n" +
+                "--------------------------------------------------------------------------------------------------------------------\n" +
+                " A5  |  T5   |VIEWSINSYSIBMIT |-INTEGER - |   N   |    0    |   4   |   0   |   0   | INTEGER  |     4     |   3   |\n" +
+                " B5  |  T5   |VIEWSINSYSIBMIT |-SMALLINT- |   Y   |    0    |   2   |   0   |   1   |SMALLINT  |     2     | NULL  |\n" +
+                " C5  |  T5   |VIEWSINSYSIBMIT |-BIGINT  - |   N   |    0    |   8   |   0   |   2   | BIGINT   |     8     |   2   |\n" +
+                " D5  |  T5   |VIEWSINSYSIBMIT |-DOUBLE  - |   Y   |    0    |   8   |   0   |   3   | DOUBLE   |     8     | NULL  |\n" +
+                " E5  |  T5   |VIEWSINSYSIBMIT |-DATE    - |   Y   |    0    |   4   |   0   |   4   |  DATE    |     4     | NULL  |\n" +
+                " F5  |  T5   |VIEWSINSYSIBMIT |-DECIMAL - |   Y   |    0    |  10   |   2   |   5   | DECIMAL  |    10     | NULL  |\n" +
+                " G5  |  T5   |VIEWSINSYSIBMIT |-TIMESTMP- |   N   |    0    |  10   |   6   |   6   |TIMESTAMP |    10     |   1   |\n" +
+                " H5  |  T5   |VIEWSINSYSIBMIT |-TIME    - |   Y   |    0    |   3   |   0   |   7   |  TIME    |     3     | NULL  |\n" +
+                " I5  |  T5   |VIEWSINSYSIBMIT |-CHAR    - |   Y   |  1252   |  10   |   0   |   8   |CHARACTER |    10     | NULL  |\n" +
+                " J5  |  T5   |VIEWSINSYSIBMIT |-VARCHAR - |   Y   |  1252   |  200  |   0   |   9   | VARCHAR  |    200    | NULL  |\n" +
+                " K5  |  T5   |VIEWSINSYSIBMIT |-BOOLEAN - |   Y   |    0    |   1   |   0   |  10   | BOOLEAN  |     1     | NULL  |\n" +
+                " L5  |  T5   |VIEWSINSYSIBMIT |-CLOB    - |   Y   |  1252   | 32765 |   0   |  11   |  CLOB    |   32765   | NULL  |\n" +
+                " M5  |  T5   |VIEWSINSYSIBMIT |-BLOB    - |   Y   |    0    |  -1   |   0   |  12   |  BLOB    |   32768   | NULL  |";
+
```

Comment:
Is it possible that the result is not sorted by colno? If so, add a order by clause can avoid intermittent IT failures

Reply:
Yes, the result may not be sorted, but the assert statement will sort the result before comparing with the expected result:
```
        Assert.assertEquals("\n" + sqlText + "\n", expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
```
So the order is deterministic.